_corporate-pages/information-security.markdown

```diff
@@ -0,0 +1,92 @@
+
+By default we redact all user-submitted personal data in incomplete or unsubmitted applications after 14 days. All user-submitted personal data in submitted applications is redacted 60 days after you have retrieved them for processing. If you feel that these retention periods do not suit you, you can choose to change them at any time.
+
+### Access to data
+
+Join Together staff are granted access to systems and data based on their role in the company or on an as-needed basis. SSO and 2FA are used to ensure access is as secure as possible.
+
+Access to user data by Join Together staff is only used to assist with support and to resolve customer issues.
+
+When working on a support issue we do our best to respect your privacy and the privacy of your members as much as possible and only access the minimum data needed to resolve your issue.
+
+### Communications Security
+
+All Join Together’s transactional emails are sent using Postmark and are encrypted using TLS, ensuring messages are encrypted in transit to remote mail servers and ISPs who support TLS.
+
+Our mail servers are configured to use [DKIM](https://postmarkapp.com/guides/dkim) and [SPF](https://postmarkapp.com/guides/spf) allowing us to control our domain’s reputation, reducing the risk of email spoofing and ensuring a high deliverability rate.
+
+## How we keep our code secure
+
+### Automatic static code analysis
+
+We  employ a continuous integration process that is automatically run against all changes made to the codebase before it can be released. This includes automated static code analysis configured to identify potential vulnerabilities in the code. We also run automated checks to alert us to vulnerabilities in any of the dependent libraries within the application.
+
+### Quality Assurance (QA)
+
+New features and code changes go through a series of stages before they are released to the live system to maintain quality and safety:
+
+- high value code changes are reviewed by a second member of staff to identify potential bugs and issues before it can be released
+- new features are shipped first to a sandboxed staging or review environment for testing and verification separately from the main production system
+- automated tests are written and maintained to ensure the system continues to operate as expected
+
+## How we secure our business
+
+### Mobile device management
+
+All hardware devices (desktops, laptops, phones) that Join Together staff use are encrypted to ensure that if stolen or lost they do not present a security risk.
+
+### Password managers and policy
+
+To ensure an acceptable level of password security, we have an existing password policy in place. Passwords that are too generic are not allowed while the use of unique passwords per website is strongly advised. We also encourage the use of password managers, for example 1Password, that help make it easier and safer for you to keep track of your credentials.
```

Comment:
Do we do anything with respect to password policy? I thought we only had a 10-char minimum

Reply:
Another good catch, I thought I'd added the weak/common password validations but clearly not yet...

Reply:
Actually, this is under the "How we secure our business" section, so it's more our internal policy around passwords, rather than the app's password restrictions.